packages/client/src/context/employee.context.tsx

```diff
@@ -16,12 +24,37 @@ export const EmployeeProvider: FC<EmployeeProviderProps> = ({ children }) => {
   const { settings, setSettings } = useSettings();
   const defaultEmployee = settings.employee ? settings.employee : null;
   const [employeeId, setEmployeeId] = useState(defaultEmployee);
+  const {
+    data: employeeData,
```

Comment:
Since this lives on a page, the page itself should call the queries and mutations